shared/receipts.mjs

```diff
@@ -0,0 +1,86 @@
+import { SKILLS_TYPES } from "./skills.mjs";
+
+class Receipt {
+  constructor({
+    id,
+    displayName,
+    createdItem,
+    quantity,
+    energyCost,
+    requiredItems,
+    requiredSkills,
+    requiredPlace,
+    durationTicks,
+    skill,
+  }) {
+    this.id = id;
+    this.displayName = displayName;
+    this.createdItem = createdItem;
+    this.quantity = quantity;
+    this.energyCost = energyCost;
+    this.requiredItems = requiredItems;
+    this.requiredSkills = requiredSkills;
+    this.requiredPlace = requiredPlace;
+    this.durationTicks = durationTicks;
+    this.skill = skill;
+  }
+}
+
+const arrowsReceipt = new Receipt({
+  id: "arrows bunch receipt",
+  displayName: "arrows bunch",
```

Comment:
It is clear that it is a receipt, also please use capital letters for displayed names.
```suggestion
  id: "arrows bunch",
  displayName: "Arrows bunch",
```